src/components/uielements/assets/assetMenu/AssetMenu.tsx

```diff
@@ -1,60 +1,55 @@
 import React, { useMemo, useCallback } from 'react'
 
-import { validBNOrZero } from '@thorchain/asgardex-util'
-import * as O from 'fp-ts/lib/Option'
+import { Asset, baseAmount } from '@thorchain/asgardex-util'
 
-import { getTickerFormat } from '../../../../helpers/stringHelper'
 import { PriceDataIndex } from '../../../../services/midgard/types'
 import { AssetPair } from '../../../../types/asgardex'
-import CoinData from '../../coins/coinData'
 import FilterMenu from '../../filterMenu'
+import AssetData from '../assetData'
 
 const filterFunction = (item: AssetPair, searchTerm: string) => {
-  const tokenName = O.toNullable(getTickerFormat(item.asset))
-  return tokenName?.toLowerCase().indexOf(searchTerm.toLowerCase()) === 0
+  const { ticker } = item.asset
+  return ticker?.toLowerCase().indexOf(searchTerm.toLowerCase()) === 0
 }
 
 type Props = {
-  asset: string
+  asset: Asset
   assetData: AssetPair[]
   priceIndex?: PriceDataIndex
-  unit: string
   searchDisable: string[]
   withSearch: boolean
   onSelect: (value: string) => void
 }
 
 const AssetMenu: React.FC<Props> = (props: Props): JSX.Element => {
-  const { assetData, asset, priceIndex = {}, unit, withSearch, searchDisable = [], onSelect = () => {} } = props
+  const { assetData, asset, priceIndex = {}, withSearch, searchDisable = [], onSelect = () => {} } = props
 
   const filteredData = useMemo(
     () =>
       assetData.filter((item) => {
-        const tokenName = O.toNullable(getTickerFormat(item.asset))
-        return tokenName?.toLowerCase() !== asset.toLowerCase()
+        const { ticker = '' } = item.asset
+        return ticker !== asset?.ticker
       }),
-    [assetData, asset]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
it seems to be useless here

Reply:
It's not, since `eslint --fix` tends to add `asset?.ticker` as a dependency instead of `asset`, which causes another `eslint` warning.